pmd-apex/src/main/resources/category/apex/bestpractices.xml

```diff
@@ -285,4 +285,31 @@ Detects when a local variable is declared and/or assigned but not used.
         </example>
     </rule>
 
+    <rule name="QueueableShouldAttachFinalizer"
```

Comment:
Maybe "QueueableWithouFinalizer" as rule name? That sounds more neutral and just expresses, what the rule detects.

```suggestion
    <rule name="QueueableWithoutFinalizer"
```